.github/actions/vcpkg-cache/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Configure vcpkg caching
+inputs:
+  id:
+    required: true
+    type: string
+runs:
+  using: composite
+  steps:
+    - name: Cache vcpkg artifacts
+      uses: actions/cache@v4
+      with:
+        path: ${{ runner.os == 'Windows' && '/.ezvcpkg' || '~/.ezvcpkg' }}
+        key: vcpkg-${{ inputs.id }}-${{ hashFiles('extern/vcpkg-overlays/**/vcpkg.json') }}-${{ hashFiles('extern/cesium-native/CMakeLists.txt', 'extern/CMakeLists.txt') }}
```

Comment:
does the caching mechanism automatically segment by OS, or is the intent to have all the different platforms / architectures in one cache blob?  The latter is going to be very tricky because as far as I know the cache upload won't merge the local packages with the overall package list.

Also, I don't know what the max cache entry size is, but trying to put all the binaries together is going to be more likely to overrun it.

Reply:
Bleh, never mind... I get that the inputs.id will do the segemenation now.